source/views/stoprint/components/error.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+
+import React from 'react'
+import type {TopLevelViewPropsType} from '../../types'
+import {View, StyleSheet, Platform} from 'react-native'
+import Icon from 'react-native-vector-icons/Ionicons'
+import {NoticeView} from '../../components/notice'
+import * as c from '../../components/colors'
+
+const ERROR_MESSAGE =
+	"Make sure you are connected to the St. Olaf WiFi \
```

Comment:
If we want to mention the St. Olaf network somewhere, I'd phrase this as `Make sure you are connected to the St. Olaf Network via eduroam or the VPN`. (WiFi is assumed, since the user is on a mobile device.)